src/libraries/System.Private.Xml/src/System/Xml/XmlConvert.cs

```diff
@@ -1008,6 +1010,8 @@ public static float ToSingle(string s)
 
         public static double ToDouble(string s)
         {
+            ArgumentNullException.ThrowIfNull(s);
```

Comment:
interestingly this currently throws NRE

Reply:
@krwq  This is to mimic current behavior. Check upper comment which shows difference when using **double/float.Parse(string...**) and **double/float.Parse(ReadOnlySpan<char>...)**  and passing **null**.

https://github.com/dotnet/runtime/blob/b40c3a3ba575babb4c0409eb5cbb9b728b1ba9c2/src/libraries/System.Private.CoreLib/src/System/Single.cs#L387-L398